File: wordgen/fictional/rook.py
from wordgen.languages import *

Rookish = Composite('rookish')
Rookish.addLanguage(English, 3)
Rookish.addLanguage(Hindi)
Rookish.addLanguage(Japanese, 2)
Rookish.addLanguage(Spanish, 2)

Elani = Composite('elani')
Elani.addLanguage(Arabic)
Elani.addLanguage(French, 3)
Elani.addLanguage(German, 2)
Elani.addLanguage(Korean)

Jas = Composite('jas')
Jas.addLanguage(Chinese, 4)
Jas.addLanguage(Afrikaner, 1)
Jas.addLanguage(Russian, 2)

Mux = Composite('mux')
Mux.addLanguage(Onomatopoetic, 20)
Mux.addLanguage(Rookish, 2)
Mux.addLanguage(Elani)
Mux.addLanguage(Jas)

File: src/hxgm30/language/syntagmata/lang/fictional/rook.clj
(ns hxgm30.language.syntagmata.lang.fictional.rook
  "To use the languages defined here, you can do the following:
  ```
    (require '[hxgm30.language.syntagmata.lang.core :as lang]
             '[hxgm30.language.syntagmata.lang.fictional.rook :as rook])
    (lang/paragraph rook/mux)
  ```
  Which will give output along the lines of the following:
  ```
    Whizz ji reira nou pebodana suran yabisebt. Dan
    aravan curassouronan mume potikertarp gun kerr
    aruppitten dep bem. Lu whoom ido ong sluckoo. Lue
    oya nyaazz hesh kedirayo. Pum settewaxth lias.
    Phes gegigupe whistramew losomepe irstendeng
    gesiweck wegetiop boo bur.
  ```")

(def rookish
  {:english 3
   :hindi 1
   :japanese 2
   :spanish 2})


(def elani
  {:arabic 1
   :french 3
   :german 2
   :korean 1})

(def jas
  {:chinese 4
   :afrikaans 1
   :russian 2})

(def mux
  (merge rookish
         elani
         jas
         {:onomatopoetic 20}))
